src/Microsoft.DocAsCode.Dfm/DfmEngineBuilder.cs

```diff
@@ -119,29 +113,43 @@ private static IMarkdownTokenRewriter InitMarkdownStyle(CompositionHost host, st
             return null;
         }
 
-        private static bool TryLoadValidatorConfig(string baseDir, MarkdownValidatorBuilder builder)
+        private static void LoadValidatorConfig(string baseDir, string templateDir, MarkdownValidatorBuilder builder)
         {
             if (string.IsNullOrEmpty(baseDir))
             {
-                return false;
+                return;
+            }
+            if (templateDir != null)
+            {
+                var configFolder = Path.Combine(templateDir, MarkdownSytleDefinition.MarkdownStyleDefinitionFolderName);
+                if (Directory.Exists(configFolder))
+                {
+                    LoadValidatorDefinition(configFolder, builder);
+                }
             }
             var configFile = Path.Combine(baseDir, MarkdownSytleConfig.MarkdownStyleFileName);
```

Comment:
no, it will be replace between multi-template.

---
In reply to: [72931437](https://github.com/dotnet/docfx/pull/545#discussion_r72931437) [](ancestors = 72931437)